src/main/java/com/campool/service/RentalService.java

```diff
@@ -74,4 +79,30 @@ private String getPolygonString(BigDecimal longitude, BigDecimal latitude, BigDe
                 + X1 + " " + Y1 + "))";
     }
 
+    public void updateStatusToRented(long rentalId, String userId) {
+        updateNewStatus(RentalStatus.TRADING, RentalStatus.RENTED, rentalId, userId);
+    }
+
+    public void completeRental(long rentalId, BookingInfo bookingInfo, String userId) {
+        if (rentalId == bookingInfo.getRentalId() && userId.equals(bookingInfo.getUserId())) {
+            updateNewStatus(RentalStatus.RENTED, RentalStatus.TRADE_COMPLETED, rentalId, userId);
+        } else {
+            throw new IllegalArgumentException("유효하지 않는 요청입니다.");
+        }
+    }
+
+    @Transactional
+    public void updateNewStatus(RentalStatus currentStatus, RentalStatus newStatus, long rentalId,
+            String userId) {
+        if (isValidRentalStatus(getRentalById(rentalId), currentStatus, userId)) {
+            rentalMapper.updateStatusById(rentalId, newStatus);
+        } else {
+            throw new IllegalStateException(currentStatus.getMessage() + "인 상태가 아닙니다.");
+        }
```

Comment:
조금 더 간결하게 표현이 가능할 것 같은데 어떠실까요?

```suggestion
        if (!isValidRentalStatus(getRentalById(rentalId), currentStatus, userId)) {
            throw new IllegalStateException(currentStatus.getMessage() + "인 상태가 아닙니다.");
        }
        
        rentalMapper.updateStatusById(rentalId, newStatus);
```

Reply:
`논리 부정 연산자`를 사용하면 막연히 가독성이 떨어질 것 같아서 사용하지 않았는데 **간결한 표현**을 생각하면 사용하는 것이 좋다는 생각이 드네요! 

리뷰 반영하겠습니다! 
감사합니다~